asv_bench/benchmarks/io/style.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+
+from pandas import DataFrame
+
+
+class RenderApply:
+
+    params = [[12, 24, 36], [12, 120, 1200]]
+    param_names = ["cols", "rows"]
+
+    def setup(self, cols, rows):
+        self.df = DataFrame(
+            np.random.randn(rows, cols),
+            columns=[f"float_{i+1}" for i in range(cols)],
+            index=[f"row_{i+1}" for i in range(rows)],
+        )
+        self._style_apply()
+
+    def time_apply(self, cols, rows):
+        # skip known performance decrease: 36x1200
```

Comment:
don't need this comment as they won't make sense to anyone who is not cognizant of this particular issue. instead you can add the issue number (or PR number) as a comment.

Reply:
I agree - I will remove it.